Ginger/GingerCore/ALM/Qtest/ExportToQtest.cs

```diff
@@ -0,0 +1,594 @@
+        {
+            try
+            {
+                BugFactory bugFactory = mTDConn.BugFactory;
+
+                //start of get bug mandatory fields
+                Customization field = (Customization)mTDConn.Customization;
+                CustomizationFields f = (CustomizationFields)field.Fields;
+
+                List BugFieldsList = f.get_Fields("BUG");
+                Dictionary<string, string> MandatoryFields = new Dictionary<string, string>();
+                Dictionary<string, List<string>> MandatoryListSelections = new Dictionary<string, List<string>>();
+                foreach (CustomizationField BugField in BugFieldsList)
+                {
+                    if (BugField.IsRequired)
+                    {
+                        List<string> FieldList = new List<string>();
+                        if (BugField.List != null && BugField.List.RootNode.Children.Count > 0)
+                        {
+                            CustomizationListNode lnode = BugField.List.RootNode;
+                            List cNodes = lnode.Children;
+                            foreach (CustomizationListNode ccNode in cNodes)
+                            {
+                                //adds list of valid selections of Field
+                                FieldList.Add(ccNode.Name);
+                            }
+                        }
+                        if (!MandatoryFields.ContainsKey(BugField.UserLabel))
+                        {
+                            MandatoryFields.Add(BugField.UserLabel, BugField.ColumnName);
+                            if (FieldList.Count > 0)
+                                MandatoryListSelections.Add(BugField.UserLabel, FieldList);
+                            else MandatoryListSelections.Add(BugField.UserLabel, null);
+                        }
+                    }
+                }
+                //end of get mandatory fileds
+                /*
+                            Bug bug = bugFactory.AddItem(System.DBNull.Value);
+                            //start of populate mandatory field with first one from dropdown
+                            foreach (dynamic item in MandatoryFields)
+                            {
+                                dynamic staticSelection = MandatoryListSelections.Where(M => M.Key == item.Key).Select(m => m.Value).FirstOrDefault();
+                                bug[item.Value] = staticSelection[0]; //statically selecting first item in list
+                            }
+                            //end of populate mandatory fields
+                */
+                Bug bug = bugFactory.AddItem(System.DBNull.Value);
+                bug.Status = Status;
+                //bug.Project = "Internal";            
+                bug.Summary = Summary;
+                bug.DetectedBy = DetectedBy;
+                bug["BG_USER_08"] = Version;
+                //bug.AssignedTo = "Nobody";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)